lms/static/sass/shared/_header.scss

```diff
@@ -349,18 +358,22 @@ header.global-new {
   box-shadow: 0 1px 5px 0 $shadow-l1;
   background: $header-bg;
 
-  nav {
+  .wrapper-header {
     @include clearfix();
     @include box-sizing(border-box);
-    width: grid-width(12);
     height: 74px;
     margin: 0 auto;
     padding: 17px 0;
+    max-width: 1180px;
```

Comment:
Are these max and mins matching something somewhere else? Seems strange to go from a flexible grid to something confined.